requirements/base.txt

```diff
@@ -12,7 +12,6 @@ gunicorn==19.7.1
 idna==2.5
 packaging==16.8
 paramiko==2.1.2
-Pillow==2.9.0
```

Comment:
Don't remove Pillow here, just bump its version to 6.2.0 and everything will be fine :D